src/Symfony/Component/Translation/Translator.php

```diff
@@ -407,7 +386,7 @@ private function loadFallbackCatalogues(string $locale): void
 
     protected function computeFallbackLocales(string $locale)
     {
-        if (null === $this->parentLocales) {
+        if (!isset($this->parentLocales)) {
             $this->parentLocales = json_decode(file_get_contents(__DIR__.'/Resources/data/parents.json'), true);
         }
```

Comment:
```suggestion
        $this->parentLocales ??= json_decode(file_get_contents(__DIR__.'/Resources/data/parents.json'), true);
```

Reply:
Committed in batch 56b1c3d110f405b279a41d2c64fbbabd3e1629e6